src/database/migrations/20190728182943-create-comment.js

```diff
@@ -0,0 +1,46 @@
+module.exports = {
+  up: (queryInterface, Sequelize) =>
```

Comment:
There seems to be two create-comment migration files here. could you remove one?